src/networking.c

```diff
@@ -2797,7 +2796,7 @@ void helloCommand(client *c) {
     }
 
     /* Let's switch to the specified RESP mode. */
-    c->resp = ver;
+    if (!!ver) c->resp = ver;
```

Comment:
i don't have anything against `!!` but in this case it's completely unnecessary. 
`!!` is used to convert a numeric value to a boolean (if you want to store only 0 or 1 into a var).
but in this case it's just an `if` that can handle any non-zero value.
```suggestion
    if (ver) c->resp = ver;
```